step-templates/wait.json

```diff
@@ -1,16 +1,20 @@
 {
-  "Id": "c69b03c9-bfa1-41ab-a712-ba74dc0512b1",
-  "Name": "Wait",
+  "Id": "03ef1bf5-4ca1-4df0-a013-6b49d9c565f6",
+  "Name": "Wait (Verbose)",
   "Description": "Pauses the process for a set duration",
   "ActionType": "Octopus.Script",
-  "Version": 0,
+  "Version": 1,
+  "CommunityActionTemplateId": null,
+  "Packages": [],
   "Properties": {
-    "Octopus.Action.Script.ScriptBody": "Start-Sleep $Seconds",
-    "Octopus.Action.Script.Syntax": "PowerShell"
+    "Octopus.Action.Script.ScriptBody": "$Wait = $Seconds\nfor ($CountDown = $Wait; $CountDown -ge 0; $CountDown--){\nWrite-Warning \"$CountDown seconds remaining\"\nStart-Sleep 1\n}",
```

Comment:
I'd suggest changing the logging from `Write-Warning "$CountDown seconds remaining"` to `Write-Verbose "$CountDown seconds remaining"` otherwise, anyone that uses this step will have the status of their deployment set to **Success with warnings** which may not be desirable.